compiler/rustc_passes/src/entry.rs

```diff
@@ -111,6 +99,7 @@ fn find_item(id: ItemId, ctxt: &mut EntryContext<'_>) {
                 });
             }
         }
+        _ => (),
```

Comment:
I'd leave this as an explicit match on `EntryPointType::None` so we can better handle new enum variants being added later.